charts/mailhog/templates/deployment.yaml

```
@@ -86,6 +86,9 @@ spec:
           {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+      {{- with .Values.extraContainers }}
+        {{- toYaml . | nindent 6 }}
```

```suggestion
        {{- toYaml . | nindent 8 }}
```
if i'm not mistaken, we need an nindent of `8` instead of `6`. With your proposal, i cannot template the mailhog chart anymore.